### ISSUE TITLE
Camera "View" button on Dashboard is no longer present

### ISSUE THREAD

**Issue by dcaud** [fruxefarms/FruxePi#46]
On the Dashboard, there used to be a button that said, I think, "View." Clicking on that would produce a current image from the camera. I found that feature very helpful. I could check whether the irrigation timer is working...although a slightly higher rez photo would make it more clear ;-). 

Was it the intent to remove that button? Can it be added back?

**Comment by fruxefarms** [comment]
The button is still there, however it only shows if the Camera is enabled. I did change the default DB settings to have all other peripherals besides the compulsory Climate (DHT22) sensor set an disabled. 

Perhaps it is a CSS issue floating off the page due to a long crop title?

A higher res photo or current photo snap might be a good idea there as well!

**Comment by dcaud** [comment]
Hmm. I don't see the button. I have the camera enabled (and I see picture auto-updates in the masthead of the main page).

I did a strong refresh in my browser (which reloads CSS) and checked it in an incognito Chrome browser, which would have forced the CSS to update.

I also checked in Firefox.

I also checked on an iPhone. 

The button is not there in any of the above browsers.

And yes, a higher res photo would be great!
